xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -157,7 +157,7 @@ This is the reason why the constructor accepts a <xref:System.Text.Json.JsonRead
         <param name="jsonData">The UTF-8 encoded JSON text to process.</param>
         <param name="isFinalBlock">
           <see langword="true" /> if the input sequence contains the entire data to process; <see langword="false" /> if the input span contains partial data with more data to follow.</param>
-        <param name="state">An object that contains the reader state. If it's the first call to the constructor, it should reflect a default state; otherwise, it should capture the state from the previous instance instance of the <see cref="T:System.Text.Json.Utf8JsonReader" /> and pass that back.</param>
+        <param name="state">A structure that contains the reader state. If it's the first call to the constructor, it should reflect a default state; otherwise, it should capture the state from the previous instance instance of the <see cref="T:System.Text.Json.Utf8JsonReader" /> by calling <see cref="T:System.Text.Json.Utf8JsonReader.CurrentState" /> and pass that back.</param>
```

Comment:
```suggestion
        <param name="state">A structure that contains the reader state. If it's the first call to the constructor, it should reflect a default state; otherwise, it should capture the state from the previous instance instance of the <see cref="T:System.Text.Json.Utf8JsonReader" /> by calling the <see cref="T:System.Text.Json.Utf8JsonReader.CurrentState" /> method and pass that back.</param>
```

Reply:
CurrentState is a property though, so should I add "property" instead?

Reply:
Good catch! The calling verb threw me off since we use that for method. Made some changes to my suggestion. Please take a look.